services/cd-service/pkg/service/overview_test.go

```diff
@@ -715,7 +715,7 @@ func TestOverviewService(t *testing.T) {
 					t.Errorf("expected two releases, got %v", test.Releases)
 				}
 				if test.Releases[1].Version != 1 {
-					t.Errorf("expected test release version to be 1, but got %d", test.Releases[0].Version)
+					t.Errorf("expected test release version to be 1, but got %d", test.Releases[1].Version)
```

Comment:
ideally we fix this in a way that prevents the same issue from happening again.
e.g.
`actual := test.Releases[1].Version`
and then use the variable in `t.Errorf`